src/components/css/react-dates@20.1.0-fix.css

```diff
@@ -0,0 +1,3 @@
+.DateInput {
+    overflow: hidden;
+}
```

Comment:
The react-dates component looks bad if the element is not very wide -- our previous usage and defaults tend to display incorrectly, proper overflow fixes this on our side for now.